insights/client/__init__.py

```diff
@@ -185,7 +185,22 @@ def _fetch(self, url, etag_file, target_path, force):
             logger.debug('Please check config, error reaching %s', url)
 
     def update(self):
+        # fetch the new eggs and gpg
         egg_paths = self.fetch()
+
```

Comment:
This block of code seems to duplicate the checks done in `verify`.

Reply:
Correct. Its more so failsafe, but I can take it out if you find it too redundant.

Reply:
Yeah, I think it's redundant enough to be removed IMHO.

Reply:
It hath been removed.